README.md

```diff
@@ -2,6 +2,18 @@
 
 > A React component for handling outside clicks
 
+## Install
+
+```shell
+$ npm install react-outside-click-handler
+```
+
+or 
+
+```shell
+$ yarn add react-outside-click-handler
```

Comment:
yarn instructions aren't necessary; anyone choosing to use a non-default package manager has already accepted the burden of knowing how to translate npm commands.

Reply:
I don't use yarn either but most of time when I add this part in the doc, people say "where is the yarn part?!" :D that's why I added this. I agree with you about this. I'll change it. :)